index.html

```diff
@@ -50,100 +40,94 @@ <h1 class="title">Isaias Del Mundo</h1>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star-half-alt"></i></td>
                     </tr>
                     <tr>
                         <td>jQuery</td>
-                        <td><i class="fas fa-star "></i></td>
-                        <td><i class="fas fa-star "></i></td>
-                        <td><i class="fas fa-star-half-alt "></i></td>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star-half-alt"></i></td>
 
                     </tr>
                     <tr>
                         <td>jQuery</td>
-                        <td><i class="fas fa-star "></i></td>
-                        <td><i class="fas fa-star "></i></td>
-                        <td><i class="fas fa-star-half-alt "></i></td>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star-half-alt"></i></td>
 
                     </tr>
                     <tr>
                         <td>WebAPIs/serverside</td>
-                        <td><i class="fas fa-star "></i></td>
-                        <td><i class="fas fa-star "></i></td>
-                        <td><i class="fas fa-star-half-alt "></i></td>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star"></i></td>
+                        <td><i class="fas fa-star-half-alt"></i></td>
                     </tr>
                 </table>
             </div>
         </div>
-        <div class="tile is-parent ">
-            <div class="tile is-child box notification is-primary ">
-                <p class="title ">Project & Hobbies</p>
-                <div class='carousel-item has-background'>
-                    <div class="title ">hangryFixed</div>
-                    <img class="hangryFixed " src="deployed-site.gif " alt=" https://idelmundo.github.io/hangryfixed/ "" width="640 " height="310 " />
+        <div class="tile is-parent">
+            <div class="tile is-child box notification is-primary">
+                <p class="title">Project & Hobbies</p>
+                <div class="carousel-item has-background">
+                    <div class="title">hangryFixed</div>
+                    <img class="hangryFixed" src="deployed-site.gif" alt="https://idelmundo.github.io/hangryfixed/" width="640" height="310" />
         </div>
-        <div class='carousel-item has-background'>
-            <div class="title ">Cars</div>
-            <img class="is-background " src="rx7.jpg " alt=" " width="640 " height="310 " />
+        <div class="carousel-item has-background">
+            <div class="title">Cars</div>
+            <img class="is-background" src="rx7.jpg" alt="" width="640" height="310" />
```

Comment:
```suggestion
            <img class="is-background" src="rx7.jpg" alt="" />
```
would be great if set the size of the image in CSS